code/game/objects/structures/lattice.dm

```diff
@@ -61,13 +60,14 @@
 	desc = "A lightweight support lattice. These hold the Justicar's station together."
 	icon = 'icons/obj/smooth_structures/lattice_clockwork.dmi'
 
-/obj/structure/lattice/clockwork/New()
+/obj/structure/lattice/clockwork/Initialize()
 	..()
 	ratvar_act()
 
 /obj/structure/lattice/clockwork/ratvar_act()
 	if(IsOdd(x+y))
-		new/obj/structure/lattice/clockwork/large(loc)
+		// this will delete the old one.
```

Comment:
please remove this comment or put it after the new thing, like `new /obj/structure/lattice/clockwork/large(loc) //this will delete the old one.`